template/template-v1.json

```diff
@@ -246,6 +246,7 @@
                 "events:*",
                 "aps:*",
                 "cloudformation:DescribeStacks",
+                "cloudformation:ListStacks",
```

Comment:
Sam comment on the other PR: I can pass the account stack ARN to you instead.

Reply:
Yes we don't need it